sql_metadb/derived_tables/invoice_lines_fund_distributions.sql

```diff
@@ -0,0 +1,49 @@
+--invoice_lines_fund_distributions
+
+DROP TABLE IF EXISTS invoice_lines_fund_distributions;
+
+CREATE TABLE invoice_lines_fund_distributions AS
+WITH funds_distr AS (
+    SELECT
+        id AS invoice_line_id,
+        jsonb_extract_path_text(dist.data, 'code') AS fund_distribution_code,
+        jsonb_extract_path_text(dist.data, 'fundId')::uuid AS fund_distribution_id,
+        jsonb_extract_path_text(dist.data, 'distributionType') AS fund_distribution_type,
+        jsonb_extract_path_text(dist.data, 'value')::numeric AS fund_distribution_value,
+        jsonb_extract_path_text(lines.jsonb, 'subTotal')::numeric(12,2) AS invoice_line_sub_total,
+        jsonb_extract_path_text(lines.jsonb, 'total')::numeric(12,2) AS invoice_line_total
```

Comment:
Please note that monetary values for derived tables in Metadb should be `numeric(19,4)`.  I will make the change here.